protocol/x/clob/ante/clob.go

```
@@ -106,6 +113,20 @@ func (cd ClobDecorator) AnteHandle(
 				return next(ctx, tx, simulate)
 			}
 
+			// HOTFIX: Ignore any short-term place orders in a transaction with a timeout height.
+			if timeoutHeight > 0 && ctx.IsCheckTx() {
+				log.InfoLog(
+					ctx,
+					"Ignored short-term place order with non-zero timeout height",
```

Nit: instead of "Ignore", "Reject"